lib/presentation/crowd_action/pages/crowd_action_details.dart

```diff
@@ -204,7 +204,7 @@ class _CrowdActionDetailsPageState extends State<CrowdActionDetailsPage> {
                     Padding(
                       padding: const EdgeInsets.fromLTRB(0, 10, 0, 10),
                       child: Text(
-                        'Short description about what the commitments are and how you can select/deselect them',
+                        'Your challenge, your rules.\n Choose which commitment you want to make this month.',
```

Comment:
Remove space after `\n`

Reply:
@Xazin Viewing the text on the app makes this look better in my opinion. That's why I added `\n` in the below code.

Reply:
![image](https://user-images.githubusercontent.com/55907631/145702555-3dfbb96d-119a-4896-842f-27217d299563.png)
![image](https://user-images.githubusercontent.com/55907631/145702568-aed2ea45-9e99-494c-91eb-8e65b22b1919.png)

Reply:
But there's no reason to have the space that comes immediately after `\n`!! Also, this text should say "Choose which commitments" with plural commitments.

Reply:
Oh 🤦🏼‍♂️ will make the necessary changes.